interfaces/Portalicious/src/app/pages/project-registrations/project-registrations.page.ts

```diff
@@ -150,19 +128,14 @@ export class ProjectRegistrationsPageComponent {
         window.open(getOriginUrl() + url, '_blank');
       },
     },
-    {
-      label: $localize`Message`,
-      icon: 'pi pi-envelope',
-      visible: this.canSendMessage(),
+    this.registrationMenuService.createContextItemForMessage({
+      projectId: this.projectId(),
       command: () => {
         this.sendMessage({
           triggeredFromContextMenu: true,
         });
       },
-    },
-    {
-      separator: true,
-    },
```

Comment:
Is there a reason the separator was removed?

Reply:
Nope :)